packages/core/src/sync-store/postgres/store.ts

```diff
@@ -824,82 +814,65 @@ export class PostgresSyncStore implements SyncStore {
     fromCheckpoint,
     toCheckpoint,
     limit,
-    logFilters = [],
-    factories = [],
+    logFilters = undefined,
+    factories = undefined,
   }: {
     fromCheckpoint: Checkpoint;
     toCheckpoint: Checkpoint;
     limit: number;
-    logFilters?: {
-      id: string;
-      chainId: number;
-      criteria: LogFilterCriteria;
-      fromBlock?: number;
-      toBlock?: number;
-      includeEventSelectors?: Hex[];
-    }[];
-    factories?: {
-      id: string;
-      chainId: number;
-      criteria: FactoryCriteria;
-      fromBlock?: number;
-      toBlock?: number;
-      includeEventSelectors?: Hex[];
-    }[];
-  }) {
-    const start = performance.now();
+  } & (
+    | {
+        logFilters: {
+          id: string;
+          chainId: number;
+          criteria: LogFilterCriteria;
+          fromBlock?: number;
+          toBlock?: number;
+          includeEventSelector: Hex;
+        }[];
+        factories: undefined;
+      }
+    | {
+        logFilters: undefined;
+        factories: {
+          id: string;
+          chainId: number;
+          criteria: FactoryCriteria;
+          fromBlock?: number;
+          toBlock?: number;
+          includeEventSelector: Hex;
+        }[];
+      }
+  )) {
+    let stopClock = startClock();
```

Comment:
I think we should pick a timing approach (either startClock or `performance.now()`) and use it everywhere in the file. And, we should use the same approach in the SQLite sync store. Wdyt?